### PR TITLE
refactor: export Resolver interface

### PR DESCRIPTION
**Pull request by danielpeintner**
fixes https://github.com/eclipse-thingweb/td-tools/issues/35

### CODE REVIEW

**Comment by danielpeintner** [comment]
Note: If nobody minds we should publish this update soon to can proceed with https://github.com/eclipse-thingweb/node-wot/pull/1312.

File: node/thing-model/package-lock.json [hunk 1]
```diff
@@ -1,12 +1,12 @@
 {
-    "name": "thing-model",
-    "version": "1.0.0",
+    "name": "@thingweb/thing-model",
+    "version": "1.0.2",
```
[the remark this answers is not in the document]
Comment (egekorkan) [reply node/thing-model/package-lock.json:3]: hmm true... not sure what really happened here

**Review by egekorkan — APPROVED** [review]
Should we go for a release?

**Comment by danielpeintner** [comment]
> Should we go for a release?

Yes. Do you or shall I a?

**Pull request merged**